Dockerfile

```
@@ -97,9 +97,9 @@ cat /opt/voicevox_engine/README.md > /dev/stderr
 exec "\$@"
 EOF
 
-ENTRYPOINT [ "/entrypoint.sh"  ]
-CMD [ "gosu", "user", "/opt/voicevox_engine/run", "--host", "0.0.0.0" ]
+ENTRYPOINT [ "/entrypoint.sh", "gosu", "user", "/opt/voicevox_engine/run" ]
```

`entrypoint.sh`は実行するコマンドを切り替える機能を実質的に持っていると思いますが、スクリプト内に実行するコマンドを埋め込んで固定した方が`entrypoint.sh`の機能が減っていいのかなと思いましたが、どうでしょう？

```suggestion
ENTRYPOINT [ "/entrypoint.sh" ]
```